frontends/web/src/routes/router.tsx

```diff
@@ -110,6 +111,18 @@ export const AppRouter = ({ devices, deviceIDs, devicesKey, accounts, activeAcco
     devices={devices}
   /></InjectParams>;
 
+  const MobileSettingsEl = <InjectParams>
+    <MobileSettings
+      deviceIDs={deviceIDs}
+    />
+  </InjectParams>;
+
+  const AppearanceEl = <InjectParams>
+    <Appearance
+      deviceIDs={deviceIDs}
+    />
+  </InjectParams>;
+
```

Comment:
@thisconnect  wdyt of this approach of passing the deviceIDs to all of the sub-pages of the settings? 

This (deviceIDs) is actually, for now anyway, only used for the tabs whether to render device setting or not. Would be cleaner if we can somehow jsut get the deviceIDs inside the `WithSettingsTab`.

Reply:
gutfeeling, the less react-router we use the easier it will be to upgrade next major version 🙈 .

I think deviceIDs should only be required for "Device settings", but probably it is also used in another tab.

Reply:
>gutfeeling, the less react-router we use the easier it will be to upgrade next major version 🙈 .
>I think deviceIDs should only be required for "Device settings", but probably it is also used in another tab.


Atm, deviceIDs are used in the `Tabs` component (new one, following the approach you suggested). And tabs are used in each subpage(s) within its wrapper `WithSettingsTabs` which gets imported in all of the (sub-)pages. So this made us require `deviceIDs` in all of those (sub-)pages...